src/test/java/thrift/logic/commands/AddIncomeCommandTest.java

```diff
@@ -48,20 +49,43 @@ public void execute_incomeAcceptedByModel_addSuccessful() {
 
     @Test
     public void undo_undoSuccessful() {
```

Comment:
Is the method name supposed to be `undoSuccessful` if it only does a successful `undo`?